main.py

```diff
@@ -0,0 +1,43 @@
+from fastapi import FastAPI, Request
+import subprocess
+import uvicorn
+
+
+app = FastAPI()
+
+
+@app.put("/download/")
+async def exec_download(request: Request):
+    data = await request.body()
+    file_name = data.decode()
+    # file = open(f"C:/Users/Elad/shest_try/files/{file_name}", 'r').read()
+    file = open(f"C:\\Users\\Elad\\shest_try\\files\\{file_name}", 'r').read()
```

Comment:
What if the file is 1000GB? Will you read it and send it at once? It would crash your RAM because you would store all the file's data in a variable.
Find a way to send it even if it is 10000GB.